backend/src/mips/services/mips.service.ts

```diff
@@ -152,12 +152,18 @@ export class MIPsService {
       if (Array.isArray(element.value)) {
         element.value.forEach(value => {
           const newValue = this.validField(element.field, value);
-          sourceInArray[`${this.searcheableField(element.field)}`] = {
-            $in: newValue,
-          };
+          if (!sourceInArray[`${this.searcheableField(element.field)}`]) {
```

Comment:
When you change something in the backend, you need to run the created tests. For that, run "npm run test". You will have to update the test related to those changes because they failed. Also, sometimes you'll need to add new test cases to cover a new flow you've added.

![image](https://user-images.githubusercontent.com/104552516/172400218-11e0852c-0dab-4989-ba16-e54ba9cff9ab.png)


but in general good job 👍👍